src/components/Sponsors/Sponsors.css

```diff
@@ -0,0 +1,64 @@
+.sponsors-container {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  margin: 50px 0px;
+}
+
+.sponsors-title {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin-bottom: 20px;
+}
+
+.box {
+  display: flex;
+  justify-content: center;
+
+  left: 35px;
+  top: 4326px;
+  margin: auto;
+  margin-bottom: 20px;
+  background-color: #a57ed4;
+  color: #fff;
+  font-size: 44px;
+  text-align: center;
+}
+
+.button {
+  flex-wrap: wrap;
+  width: 556px;
+  left: 472px;
+  top: 4562px;
+
+  background: #ebebeb;
+  border-radius: 25px;
+  font-size: 35px;
+}
+.sponsors-row {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
```

Comment:
I noticed that you included `justify-content` and `flex-wrap` in a lot of places ~ do you think you can review where you've included them and check if they're needed or not?